{"name": "Verify that user fails to login specific repository successfully via Dashboard login page with incorrect credentials", "status": "passed", "steps": [{"name": "Login to system", "status": "passed", "parameters": [{"name": "username", "value": "'abc'"}, {"name": "password", "value": "'abc'"}], "start": 1722250434969, "stop": 1722250435582}, {"name": "Accept The Alert", "status": "passed", "start": 1722250436747, "stop": 1722250446782}], "attachments": [{"name": "stdout", "source": "08449a99-efb1-4db2-ba8e-a8f4f357a74c-attachment.txt", "type": "text/plain"}], "start": 1722250434969, "stop": 1722250446782, "uuid": "811ba8fa-0611-4410-b558-9c30de6b4cc7", "historyId": "f7d89d05a4f00bc25635fa7073cf2d6d", "testCaseId": "f7d89d05a4f00bc25635fa7073cf2d6d", "fullName": "tests.test_DA_LOGIN.test_DA_LOGIN#test_DA_LOGIN_TC002_1___abc____abc__", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_DA_LOGIN"}, {"name": "subSuite", "value": "test_DA_LOGIN"}, {"name": "host", "value": "AGD14040"}, {"name": "thread", "value": "14568-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_DA_LOGIN"}]}